tests/integration/records/test_records_views.py

```diff
@@ -24,22 +24,33 @@
 
 import json
 
-from factories.db.invenio_records import TestRecordMetadata
+from invenio_search import current_search_client as es
+
+from inspirehep.modules.records.api import InspireRecord
 
 
 def test_literature_citations_api_with_results(isolated_api_client):
     record_json = {
+        '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'document_type': [
+            'article',
+        ],
         'control_number': 111,
         'titles': [
             {
                 'title': 'Jessica Jones',
             },
         ],
+        '_collections': ['Literature']
     }
-    record = TestRecordMetadata.create_from_kwargs(
-        json=record_json).inspire_record
+    record = InspireRecord.create(record_json)
```

Comment:
by using ``TestRecordMetadata`` the record is indexed, is not enough for our test cases?

Reply:
I wanted to use TestRecordMetadata at first but then I looked through the other tests that we have and that use elastic search and they did it like this so I wanted to keep it consistent with what we had so far.

Reply:
The other tests are might be old, we are trying to use ``TestRecordMetadata`` in the new tests and while we refactoring.